app/assets/javascripts/govuk/selection-buttons.js

```diff
@@ -6,8 +6,6 @@
   if (typeof GOVUK === 'undefined') { root.GOVUK = {}; }
 
   var SelectionButtons = function (elmsOrSelector, opts) {
-    var $elms;
```

Comment:
sorry, its not clear but this code doesnt originate in the kit - I think it comes from the Frontend toolkit? Or Elements? @gemmaleigh ?